src/components/pager/pager.style.ts

```diff
@@ -9,7 +9,7 @@ import Link from "../link";
 
 const StyledSelectContainer = styled.div`
   height: 26px;
-  width: 55px;
+  width: 64px;
```

Comment:
question: why has 64px been chosen specifically?

On the surface this seem to be a bit of an arbitrary number to choose. Does it definitely require us using a fixed width? Happy to go with what you deem best, but just wanted to double check.

Reply:
I think we could maybe do it using grid to adjust it dynamically, but sure, we'll stick with a fixed width for now. No point complicating it at this juncture. Approved.